cashu/mint/ledger.py

```diff
@@ -941,8 +942,8 @@ async def melt(
                 f"status: {status}, preimage: {preimage}, fee_msat: {fee_msat}"
             )
 
-            if status == True:
-                logger.trace(f"invalidating proofs")
+            if status is True:

```

Comment:
Nitcpick: 
```suggestion
            if status:
```
Unless we *really* care that it **is** `True` rather than truthy, just `if status:` is fine